src/GitHubActions.jl

```diff
@@ -213,14 +213,33 @@ function set_failed(msg)
 end
 
 """
-    GitHubActionsLogger()
+    GitHubActionsLogger(level)
 
 A logger that prints to standard output in the format expected by GitHub Actions.
+
+By default, GitHubActionsLogger inspects the
+[`RUNNER_DEBUG` environment variable](https://docs.github.com/en/actions/learn-github-actions/variables)
+to match the log-level to that set in GitHub Actions.
 """
-struct GitHubActionsLogger <: AbstractLogger end
+struct GitHubActionsLogger <: AbstractLogger
+    level::LogLevel
+end
```

Comment:
```suggestion
Base.@kwdef struct GitHubActionsLogger <: AbstractLogger
    level::LogLevel = Debug
end
```

I think without this the change is breaking?

Reply:
Oops, this isn't correct as this clobbers the positional constructor. But I think we need `GitHubActionsLogger() = GitHubActionsLogger(Debug)`.

Reply:
I made a zero-arg constructor just below